src/modeler/AutoRest.Swagger/Model/SwaggerBase.cs

```diff
@@ -24,6 +24,8 @@ public SwaggerBase()
         [CollectionRule(typeof(NonEmptyClientName))]
         [CollectionRule(typeof(NextLinkPropertyMustExist))]
         [CollectionRule(typeof(PageableRequires200Response))]
+        [CollectionRule(typeof(MutabilityValidValuesRule))]
```

Comment:
[comment, no action required] We are surely going to hell... we're putting a ton of rules on the extensions collection. We're really going to have to think this thru better in AutoRest 2.0 //cc @olydis